airbyte-integrations/connectors/source-klaviyo/source_klaviyo/manifest.yaml

```diff
@@ -0,0 +1,365 @@
+      path: "lists/"
+
+  metrics_stream:
+    type: DeclarativeStream
+    name: Metrics
+    primary_key:
+      - id
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      paginator:
+        $ref: "#/definitions/cursor_paginator"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          fields[metric]: '{{ config[''fields_metric_''] }}'
+          filter: '{{ config[''filter''] }}'
+        request_headers:
+          revision: '{{ config[''revision''] }}'
+    $parameters:
+      name: "metrics"
+      path: "metrics/"
+
+  flows_stream:
+    type: DeclarativeStream
+    name: Flows
+    primary_key:
+      - id
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      paginator:
+        $ref: "#/definitions/cursor_with_page_size_paginator"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          fields[flow-action]: '{{ config[''fields_flow_action_''] }}'
+          fields[flow]: '{{ config[''fields_flow_''] }}'
+          fields[tag]: '{{ config[''fields_tag_''] }}'
+          filter: '{{ config[''filter''] }}'
+          include: '{{ config[''include''] }}'
+          sort: '{{ config[''sort''] }}'
+        request_headers:
+          revision: '{{ config[''revision''] }}'
+    $parameters:
+      name: "flows"
+      path: "flows/"
+      page_size: 50
+
+  profiles_stream:
+    type: DeclarativeStream
+    name: Profiles
+    primary_key:
+      - id
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      paginator:
+        $ref: "#/definitions/cursor_with_page_size_paginator"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          additional-fields[profile]: '{{ config[''additional_fields_profile_''] }}'
+          fields[profile]: '{{ config[''fields_profile_''] }}'
+          filter: '{{ config[''filter''] }}'
+          sort: '{{ config[''sort''] }}'
+        request_headers:
+          revision: '{{ config[''revision''] }}'
+    $parameters:
+      name: "profiles"
+      path: "profiles/"
+      page_size: 20
+
+# Streams
+streams:
+  - "#/definitions/campaigns_stream"
+  - "#/definitions/events_stream"
+  - "#/definitions/globalexclusions_stream"
+  - "#/definitions/lists_stream"
+  - "#/definitions/metrics_stream"
+  - "#/definitions/flows_stream"
+  - "#/definitions/profiles_stream"
+
+# Check
+check:
+  type: CheckStream
+  stream_names:
+    - Campaigns
+
+# Connection Specification
+spec:
+  connection_specification:
+    $schema: http://json-schema.org/draft-07/schema#
+    type: object
+    required:
+      - api_key
+      - filter
+      - revision
+    properties:
+      api_key:
+        type: string
+        title: API Key
+        airbyte_secret: true
+        order: 0
```

Comment:
@topefolorunso For this hackathon the request is to move the connector from Python CDK to low-code. You're introducing new parameters in the spec for Klaviyo connector and this is outside of the scope of the contribution. You're more than welcome to introduce this in a next contribution but for now please remove it.

Reply:
The new parameters are based on the updated API. The API used in the Python CDK is scheduled to be end-of-lifed on Jan 1, 2024 as per https://developers.klaviyo.com/en/v1-2/reference/api-overview

I will remove them as requested and introduce the changes in a new contribution

Reply:
I appreciate the effort @topefolorunso but moving the connector to low code already is a big change. updating the API will introduce a lot more to review and ensure everything is working.

Reply:
Oh, I understand that. I have made the requested changes now. Kindly review.